### ISSUE TITLE
[ui5-dynamic-page]: can not updated "expanded header region" for accessibility

### ISSUE THREAD

**Issue by raoakash01** [SAP/ui5-webcomponents#11948]
### Bug Description

We are not able to update the region name for the "expanded header region". We have reported with an accessibility ticket that this name should be more descriptive.

### Expected Behaviour

<img width="954" height="415" alt="Image" src="https://github.com/user-attachments/assets/29e7c50f-1124-4d86-9525-e442522b25ee" />

### Isolated Example

https://stackblitz.com/edit/github-sidvower?file=src%2FApp.tsx

### Priority

High

### UI5 Web Components Version

2.4.0

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by NHristov-sap** [comment]
Hello @SAP/ui5-webcomponents-topic-p  ,

Please check this issue for Dynamic Page component.

Best Regards,
Nikolay Hristov

**Comment by yanaminkova** [comment]
Hello @raoakash01,

the `ui5-dynamic-page` component follows the screen reader specification". For example, when the header is expanded, the screen reader announces “Header expanded region,” which is the expected behavior.
"Dynamic Page`s Snapping Header: role: “region”, aria-expanded: “true/false”, aria-label: “Header expanded/ Header snapped’." (This is copied directly from the accessibility specification.) 
Therefore, I do not see an issue here. 

Could you please provide a bit more information about what you expect to be announced when the header is expanded?

Regards,
Yana

**Comment by raoakash01** [comment]
hey @yanaminkova 
It announces “Header expanded region" but we got a costumer ticket that this announcement should be more specific or descriptive 
for example - 'title of the page Header expanded region'

Regards,
Akash

**Comment by yanaminkova** [comment]
Hi @raoakash01,

Since the control has been implemented according to the specification, your request is considered a feature request. Based on the new design requirements, this item will need to go through the InnerSource process.
Could you please create a Jira ticket for Central Design and include the E1 reference here?

Thanks and Regards,
Yana

**Comment by raoakash01** [comment]
hey @yanaminkova 
I have raised request 
Jira Ticket :- [SAPDESIGNSYSTEM-334]

What is this E1 reference number!

**Comment by dobrinyonkov** [comment]
Hi @raoakash01,

the SAPDESIGNSYSTEM id is enogh, thanks! 
Design team is now looking into this request.

**Comment by wilson-xu-sap** [comment]
Hi Any update on this?

**Comment by petyabegovska** [comment]
Hi @wilson-xu-sap,

There is a Jira item in SAP Design: FIORIDESIGN-15733 planned for Q1 and it's still open.

Best regards,
Petya

**Comment by wilson-xu-sap** [comment]
Hi @petyabegovska Thanks. What about now?
